sceptre/stack.py

```diff
@@ -520,28 +520,57 @@ def create_change_set(self, change_set_name):
 
         :param change_set_name: The name of the change set.
         :type change_set_name: str
+        :raises: botocore.exceptions.ClientError
         """
-        create_change_set_kwargs = {
-            "StackName": self.external_name,
-            "Parameters": self._format_parameters(self.parameters),
-            "Capabilities": ['CAPABILITY_IAM', 'CAPABILITY_NAMED_IAM'],
-            "ChangeSetName": change_set_name,
-            "NotificationARNs": self.config.get("notifications", []),
-            "Tags": [
-                {"Key": str(k), "Value": str(v)}
-                for k, v in self.config.get("stack_tags", {}).items()
-            ]
-        }
-        create_change_set_kwargs.update(self._get_template_details())
-        create_change_set_kwargs.update(self._get_role_arn())
-        self.logger.debug(
-            "%s - Creating change set '%s'", self.name, change_set_name
-        )
-        self.connection_manager.call(
-            service="cloudformation",
-            command="create_change_set",
-            kwargs=create_change_set_kwargs
-        )
+        for change_set_type in ["UPDATE", "CREATE"]:
+            create_change_set_kwargs = {
+                "StackName": self.external_name,
+                "Parameters": self._format_parameters(self.parameters),
+                "Capabilities": ['CAPABILITY_IAM', 'CAPABILITY_NAMED_IAM'],
+                "ChangeSetName": change_set_name,
+                "NotificationARNs": self.config.get("notifications", []),
+                "ChangeSetType": change_set_type,
+                "Tags": [
+                    {"Key": str(k), "Value": str(v)}
+                    for k, v in self.config.get("stack_tags", {}).items()
+                ]
+            }
+            create_change_set_kwargs.update(self._get_template_details())
+            create_change_set_kwargs.update(self._get_role_arn())
+            self.logger.debug(
+                "%s - Creating %s change set '%s'",
+                self.name, change_set_type, change_set_name
+            )
+            try:
+                self.connection_manager.call(
+                    service="cloudformation",
+                    command="create_change_set",
+                    kwargs=create_change_set_kwargs
+                )
+            except botocore.exceptions.ClientError as e:
+                # If a stack does not exist, describe_stacks will throw an
+                # exception. Unfortunately we don't have a better way than
+                # parsing the exception msg to understand the nature of this
+                # exception.
+                msg = str(e)
+
+                if (
+                        "Stack with id {0} does not exist".format(
+                            self.external_name)
+                        in msg
+                ):
+                    self.logger.debug(
```

Comment:
Confusing debug message - not sure the `for` loop is the best way to go about it.... you could just have the `create_change_set` call instead of the `continue`?